### PR TITLE
impact-mapping-first-edit

### PR DESCRIPTION
**Pull request by nmuller66**

### CODE REVIEW

File: docs/slides/content/impact-mapping.md [hunk 1]
```diff
@@ -54,9 +54,9 @@ At the end of the Impact Mapping exercise, you should have:
 </div>
 <div class="col" data-markdown>
 
-_It's as a lightweight, collaborative planning technique for teams that want to make a big impact with software products_
+_A lightweight, collaborative planning technique for teams that want to make a big impact with software products_.
```
Comment (nmuller66) [inline docs/slides/content/impact-mapping.md:57]:
```suggestion
_Impact mapping is a lightweight, collaborative planning technique for teams that want to make a big impact with software products_.
```

File: docs/slides/content/impact-mapping.md [hunk 2]
```diff
@@ -54,9 +54,9 @@ At the end of the Impact Mapping exercise, you should have:
...
 
-_It help delivery teams and stakeholders visualize roadmaps, explain how deliverables connect to user needs, and communicate how user outcomes relate to higher level organizational goals_
+_Helps delivery teams and stakeholders visualize roadmaps, explain how deliverables connect to user needs, and communicate how user outcomes relate to higher level organizational goals_.
```
Comment (nmuller66) [inline docs/slides/content/impact-mapping.md:59]:
```suggestion
_It helps delivery teams and stakeholders visualize roadmaps, explain how deliverables connect to user needs, and communicate how user outcomes relate to higher-level organizational goals_.
```

**Comment by nmuller66** [comment]
There are two edits required to images: 

The Impact Map-World Health Organization board goal is not clear or correct. I'd suggest 
"A global, unified platform for learning, with rich user experiences that support integration and data analytics, enabling a state-of-the-art training institution"

The first impact sticky is a fragment and needs to be edited. I'd suggest:
Increase Collaboration to bring together supporting technologies"

**Comment by ckavili** [comment]
LGTM @nmuller66! I fixed the typo on the sticky.

**Pull request merged**